zonemaster/zonemaster-gui
# add default option for protocol setting

## Purpose

This way the default settings on the GUI will use the default on the profile with the ability to override those settings.

## Context

This is a proposal following discussion on #217.

## How to test this PR

* Using the default setting on a dual stack backend should not change anything.
* Disabling one protocol should disable the tests using that protocol, could be checked in test results.
* Testing a domain using backend that only support one protocol (ex. zonemaster.ppsfleet.navy ) with the default settings should only trigger the tests of the supported protocol.

![Screenshot 2021-06-14 at 16-36-46 Zonemaster](https://user-images.githubusercontent.com/19394895/121910022-c60f1900-cd2e-11eb-9fa0-a1298de229a3.png)
